docs/infrastructure.md

```diff
@@ -0,0 +1,87 @@
+## Infrastructure
+
+### [General Deployment Process Overview Doc](https://github.com/broadinstitute/dsp-devops-wiki/wiki/deployment_process)
```

Comment:
Is that document publicly available? If/when we have partners, will they need to read it?

Reply:
yep sure is